packages/components/src/components/step/step.ts

```diff
@@ -37,19 +41,22 @@ export default class SdStep extends SolidElement {
   @property({ reflect: true }) orientation: 'horizontal' | 'vertical' = 'horizontal';
 
   /** Determines the state of the step. */
-  @property({ reflect: true }) state: 'waiting' | 'inProgress' | 'finished' = 'waiting';
+  @property({ reflect: true }) state: 'disabled' | 'current' | 'default' = 'default';
```

Comment:
Looks great in general – just one note here coming from the investigation: https://github.com/solid-design-system/solid/pull/1007#discussion_r1592590947

Instead of "state" with options, introduce `current` and `disabled` as booleans, so it fits to our other components.